spec/ttml1.xml

```diff
@@ -4594,26 +4594,26 @@ as a style property only to those element types indicated in the following table
 </tr>
 </tbody>
 </table>
-<p>If the value of this attribute is <code>normal</code>, then the
-computed value of the style property must be considered to be no less than the
+<p>If the computed value of this attribute is <code>normal</code>, then the
+used value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
 <specref ref="semantics-region-layout-step-1"/>.</p>
 <note role="elaboration">
-<p>It is recommended that presentation processors use a computed line height that is
-125% of the size of the largest applicable font size when the value <code>normal</code>
-is specified. The choice of 125% allows use of a font size of <code>0.8c</code> in order
+<p>If the computed value of this attribute is <code>normal</code>, it is recommended that presentation processors
+use a line height that is 125% of the size of the largest applicable font size.
+The choice of 125% allows use of a font size of <code>0.8c</code> in order
 to obtain a line height of <code>1c</code>.</p>
 </note>
 <p>If specified as a <loc
 href="#style-value-length">&lt;length&gt;</loc>, then the length must
 be non-negative.</p>
-<p>If a computed value of the property associated with this attribute is not supported,
+<p>If the used value of the property associated with this attribute is not supported,
```

Comment:
Could reword in terms of the actual value, since it looks odd to say "if the used value is not supported, use a different value" (both are expressed in terms of "use", whereas in fact it's the mapping from used value to actual value that is being affected, in CSS terms)